plugins/templates/rules/jms.j2

```diff
@@ -0,0 +1,15 @@
+{ "messaging" => {
+    "hornetq-server" => {
+      "default" => {
+        "jms-queue" => {
+          {% for item in values %}
+            "{{ item['queue_address'] }}" => {
+            "entries" => [{% for entry in item['entries'] %}"{{ entry }}"{{ "," if not loop.last }}{% endfor %}],
```

Comment:
That's exactly the change I did locally to try to make it work, but when I passed entries as list, the double quote around {{ entry }} are being replaced by single quote making it an invalid array for jcliff. However, I did not try to that in one-liner (I'll check now if it somehow help).

Reply:
@sabre1041 can you share the playbook? I'll be interested to see how you set the value of entries?